wiki/advantages-and-features.md

```diff
@@ -0,0 +1,185 @@
+  
+(note you can earn both Gridcoin and OBytes at the same time)
+
+
+#### Why's Gridcoin Different?
+
+* OByte's main focus is different from Gridcoin 
+    * Science aspect is a secondary or tertiary goal
+        * Fewer OBytes sent to reward science and more goes to other distribution methods
+    
+    * For Gridcoin science is a primary goal
+        * Larger percentage rewards science 
+
+^
+^
+
+* OByte's bots are heavily centralized 
+    * Happening through one bot that could stop at any point
+        * Funding to this bot comes entirely through a grant from a fund 
+          controlled currently by 4 people
+
+    * In Gridcoin, coins are distributed as new coins in the network not through a single fund or bot
+
+^
+^
+
+* OByte only rewards World Community Grid and no other BOINC project
+    * On any other project, you can't earn OBytes for it
+
+    * With Gridcoin there are 18 project you can contribute to (as of 2020-08-08)
+
+^
+^
+
+
+### Comparison to Primecoin
+---
+#### What's Primecoin
+Primecoin is a crytpocurrency that uses a proof of work algorithm to 
+find [Cunningham chains](https://en.wikipedia.org/wiki/Cunningham_chain) of prime numbers.
+
+
+#### Why's Gridcoin Different?
+* Primecoin can only find prime numbers and do no scientific computation 
+    * Limited by Proof of Work algorithm 
+        * Any expansion of Primecoin to other work would need
+          to be a Proof of Work algorithm which need abilities that many
+          scientific problems lack 
+            * See the challenges with even using primes in [Primecoin's whitepaper](https://primecoin.io/bin/primecoin-paper.pdf)
+
+    * With Gridcoin more can be rewarded such as fighting disease, finding 
+      asteroids, and much more
+
+^
+^
+
+*  Gridcoin has more active development and community
+    * Comparison of development activity in the last month:
+        * [Gridcoin's here](https://github.com/gridcoin-community/Gridcoin-Research/pulse/monthly) 
+        * [Primecoin's here](https://github.com/primecoin/primecoin/pulse/monthly)
+
+### Comparison to Tokens
+---
+#### What Are These Tokens
+There are a variety of tokens that have tried/are trying to reward scientific 
+work. 
+
+#### Why's Gridcoin Different
+* Most of these tokens are heavily centralized 
+    * One person/group decides how they should be distributed
+        * Many of these tokens have suddenly stopped because the people behind 
+          the projects decided to stop giving tokens out.
+
+    * Gridcoin has systems in place to not give one person or group the 
+      ability to control distribution. 
+        * The [Scrapers](scraper "wikilink"), Proof of Stake, and more allow for 
+          a much more decentralized system.
+
+^
+^
+
+* New tokens lack an existing community
+    * Have to spend even more time on marketing, outreach, getting more 
+    developers, etc.
+        * Chicken and egg problem
```

Comment:
not explained

Reply:
Is this better `Have to overcome the chicken and egg problem for adoption`?